modules/mlflow/variables.tf

```diff
@@ -10,6 +10,12 @@ variable "create_model" {
   default     = true
 }
 
+variable "enable_mlflow_nodeport" {
+  description = "Boolean value that enables the NodePort service for MLFlow"
```

Comment:
suggestion: what about ditching enable from the name and using mlflow_server_nodeport since that's the name used for the charm during deployment?

Reply:
I think it would be confused as the value for the port rather than the boolean, this also depends what we decide in https://github.com/canonical/charmed-mlflow-solutions/pull/9#discussion_r1936865581